test/smoke_test.sh

```diff
@@ -23,10 +23,81 @@ function get_logs_clean_and_exit() {
   eval "$COMPOSE_CMD down"
 }
 
+function check_mandatory_vars() {
+
+    declare -a mandatory
+    declare -a allure_mandatory
+
+    if [[ "${CI:-false}" == "true" ]]; then
+      mandatory=(
+        WALLARM_API_TOKEN
+        WALLARM_API_HOST
+        WALLARM_API_PRESET
+        CLIENT_ID
+        USER_TOKEN
+        WEBHOOK_API_KEY
+        WEBHOOK_UUID
+        PYTEST_WORKERS
+        PYTEST_ARGS
+      )
+    else
+      mandatory=(
+        WALLARM_API_TOKEN
+        WALLARM_API_HOST
+        WALLARM_API_PRESET
+        CLIENT_ID
+        USER_TOKEN
+        SMOKE_REGISTRY_TOKEN
```

Comment:
Can we avoid such duplication and check registry creds variables separately? Same as we check allure variables

Reply:
yes we can.